frontend/src/entry/model/mirroredModels/ReleasesSelector.tsx

```diff
@@ -0,0 +1,67 @@
+import DeleteIcon from '@mui/icons-material/Delete'
+import { Checkbox, FormControlLabel, Grid, IconButton, Tooltip, Typography } from '@mui/material'
+import { useGetReleasesForModelId } from 'actions/release'
+import { ChangeEvent, useCallback, useMemo, useState } from 'react'
+import Loading from 'src/common/Loading'
+import ReleaseDisplay from 'src/entry/model/releases/ReleaseDisplay'
+import MessageAlert from 'src/MessageAlert'
+import { EntryInterface, ReleaseInterface } from 'types/types'
+
+type ReleasesSelectorProps = {
+  model: EntryInterface
+  selectedReleases: ReleaseInterface[]
+  setSelectedReleases: (values: ReleaseInterface[]) => void
```

Comment:
I should've picked this up earlier, but using `set...` as a variable name should be reserved for `useState` setters. I know that's effectively what this is based on your implementation, but as far as this component knows it could be anything. So we shouldn't assume anything and as such I wouldn't name it as a setter.

Instead I would go with the convention of treating this as an event. We denote these by `on...`. So in this case it would be `onUpdateSelectedReleases`.